libraries/ms-common/inc/ms_test_helper_datagram.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+/* Test helpers for datagrams
+ */
+
+#include "bootloader_can.h"
+#include "can_datagram.h"
+#include "ms_test_helpers.h"
+#include "status.h"
+
+#define DATAGRAM_PROCESS_ALL(e)                                 \
+  while (can_datagram_get_status() == DATAGRAM_STATUS_ACTIVE) { \
+    MS_TEST_HELPER_AWAIT_EVENT(e);                              \
+    can_datagram_process_event(&e);                             \
+    can_process_event(&e);                                      \
+  }
+
+// initialize the datagram helper and dependencies
+void init_datagram_helper(CanStorage *can_storage, CanSettings *can_settings, uint8_t board_id,
```

Comment:
The usual naming convention would be `ms_test_helper_datagram_init` just to make it clear what module it's from.